frontend-app/src/Thermometer/TemperatureControl.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+// import './TemperatureControl.css';
+import './TemperatureControl.css'
+
+function TemperatureControl(){
+
+    const [temp, setTemp] = useState(10);
+
+    const rise = () =>{
+        temp < 30 && setTemp(temp + 1);
+    }
+
+    const drop =() =>{
+        temp > 0 && setTemp(temp - 1);
+    }
+
+    const myClass = temp >= 15 ? " hot":" cold";
+
+    return (
+        <div className="app-container">
+          <div className="temperature-display-container">
+            <div className={"temperature-display" + myClass}  >
```

Comment:
предлагаю вот так сделать?
```suggestion
            <div className={`temperature-display ${temp >= 15 ? "hot" : "cold"}`}  >
```